### ISSUE TITLE
Use ul element

### ISSUE THREAD

**Issue by LJWatson** [w3c/Unitas#44]
Where ```<li>``` elements are used, they need to be children of a ```<ul>``` element. For example on the "All groups" page (and others that share the same template pattern):
```
<div class="panel-body">
<li class="list-group-item"><a title="Accessibility in India Community Group" href="?g=61068">Accessibility in India Community Group</a></li>
...
</div>
```
Without the parent ```<ul>``` element, the browser does not expose the ```<li>``` elements as being part of a collection. This means that people who use a screen reader (because they cannot see the page) are unaware of how big the collection is - useful metadata that is otherwise obvious at a glance.

**Comment by tripu** [comment]
Spot on. I didn't realise we had orphan `li`'s like that&hellip;
